### ISSUE TITLE
Doublebracket-Syntax doesn't work for iteration items

### ISSUE THREAD

**Issue by opensource21** [thymeleaf/thymeleaf#275]
If you have an Object Task with an attribut budget and declare via annotation for budget a special formatter, I get the following surprising behavior:

```
<span th:text=${{task.budget}}></span>
```

works fine, but

```
 <ul th:each="task:${tasks}>
      <li th:text=${{task.budget}}></li>
 <ul>
```

doesn't work. This is an extremly surprising behavior.

**Comment by danielfernandez** [comment]
This is known behaviour. Please have a look at thymeleaf/thymeleaf-spring#37 and thymeleaf/thymeleaf-spring#49.

I'm closing this ticket as this belongs to the thymeleaf-spring repository, and the topic is already covered in the referenced tickets.

Thanks.

**Comment by opensource21** [comment]
Thanks!

**Comment by danielfernandez** [comment]
Note this has been implemented and will be a part of Thymeleaf 2.1.3. See thymeleaf/thymeleaf-spring#49 for more information.
